### PR TITLE
Changed context and config to run with newest nearcore

### PR DESCRIPTION
**Pull request by willemneal**

### CODE REVIEW

File: tests/test.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+rm /tmp/main.wasm
```
Comment (janedegtiareva) [inline tests/test.sh]: I was planning for this to be a temporary file until I can find a nicer way to call these tests. Do you think it's worth while still including it in the branch?

**Comment by janedegtiareva** [comment]
Could you rebase it to merge to staging instead? looks good after that.

File: tests/test.sh (hunk 1 above)
Comment (willemneal) [reply tests/test.sh]: I think it's a good starting place to add the tests and set up running nearcore, perhaps using docker?

**Comment by willemneal** [comment]
Cool then I'll close this for now and open up a new one.

**Pull request closed without merging**